core/components/atoms/collapsible/__stories__/index.story.jsx

```diff
@@ -8,30 +8,34 @@ export const all = () => {
   const [expanded, setExpanded] = React.useState(false);
 
   return (
-    <Collapsible expanded={expanded} hoverable={hoverable} height="100vh" onToggle={setExpanded}>
-      <div className="d-flex pt-4">
-        <Icon name="events" className="d-flex align-items-center px-6 Text--regular" />
-        {expanded && <Text className="mr-6">Collapsible</Text>}
-      </div>
-    </Collapsible>
+    <div>
```

Comment:
please check this story once, look like it is broken.
![Uploading Screenshot 2025-05-07 at 1.00.54 PM.png…]()